privacyidea/lib/importotp.py

```diff
@@ -537,10 +537,11 @@ def parsePSKCdata(xml_data,
             # Check if hashlib is explicitly set in file
             if parameters.suite and parameters.suite.string:
                 hash_lib = parameters.suite.string.lower()
-            token["hashlib"] = hash_lib
         except Exception as e:
             log.warning(f"No compatible suite contained, falling back to default {hash_lib}.")
```

Comment:
I am so sorry. I only saw this now.

The f-format will not work.

Please use the u-format 

   u"....some test {0!s}".format(hash_lib)